src/test/FreeC/Frontend/Haskell/SimplifierTests.hs

```diff
@@ -0,0 +1,45 @@
+-- | This module contains tests for "FreeC.Frontend.Haskell.Simplifier".
+module FreeC.Frontend.Haskell.SimplifierTests ( testSimplifier ) where
+
+import           Control.Monad                     ( (>=>) )
+import           Test.Hspec
+
+import           FreeC.Frontend.Haskell.Parser
+import           FreeC.Frontend.Haskell.Simplifier
+import           FreeC.IR.SrcSpan
+import qualified FreeC.IR.Syntax                   as IR
+import           FreeC.Monad.Class.Testable
+import           FreeC.Test.Expectations
+import           FreeC.Test.Parser
+
+-------------------------------------------------------------------------------
+-- Utility functions                                                         --
+-------------------------------------------------------------------------------
+-- | Parses a Haskell expression and converts it to IR.
+parseAndSimplifyExpr :: String -> Simplifier IR.Expr
+parseAndSimplifyExpr = (parseHaskell . mkSrcFile "<test-input>")
+  >=> simplifyExpr
+
+-- | Parses the given Haskell and IR expressions, converts the Haskell
+--   expression to IR .
```

Comment:
This seems incomplete:
```suggestion
--   expression to IR and sets the expectation that the given IR expression is
--   produced.
```
At the very least, the space after `IR` has to be removed.